etna/datasets/tsdataset.py

```diff
@@ -857,6 +868,45 @@ def train_test_split(
 
         return train, test
 
+    def update_columns_from_pandas(self, df: pd.DataFrame, regressors: Optional[List[str]] = None):
+        """Update the dataset with the new columns from pandas dataframe.
+
+        Parameters
+        ----------
+        df:
+            Dataframe with the new columns in wide etna format
+            If columns with the same names already exist in the dataset, there values will be updated
```

Comment:
Looks a little bit strange: `If columns with the same names already exist in the dataset, there values will be updated`.
May be: `If columns with the same names already exist in the dataset, then values will be updated` ?

Reply:
fixed